src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -444,10 +485,24 @@ JitConstants FullyConnected_bf_tiled::GetJitConstants(const fully_connected_para
         jit.AddConstant(MakeJitConstant("FILTER_LOAD_BLOCK_SIZE", block_read_size));
         jit.AddConstant(MakeJitConstant("FILTER_ELEMENTS_PER_LOAD", weights_elements_per_load));
         jit.Merge(make_int4_packed_type_jit_constant("INT4_PACKED_TYPE_PRELOAD", params.weights.GetDType(), weights_elements_per_load));
+    } else {
+        jit.AddConstant(MakeJitConstant("USE_SLM", 0));
+    }
+
+    // Validated perf gain, Dynamic quantize force enable SCALE_POST_OP for char type multiplication
```

Comment:
isn't it better to put it inside of `if (dispatchData.use_slm) {` so that reader can explicitly understand that dynamic quantization is supported only from `use_slm` case?